hack/observability/grafana/dashboards/cluster-api.json

```diff
@@ -557,514 +557,613 @@
       "type": "stat"
     },
     {
-      "collapsed": false,
+      "collapsed": true,
```

Comment:
Just changed it everywhere that all rows are collapsed per default. This just avoids a bit of unnecessary metrics loading when the dashboard is opened